packages/block-library/src/tag-cloud/index.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Server-side rendering of the `core/tag-cloud` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the `core/tag-cloud` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the tag cloud for selected taxonomy.
+ */
+function render_block_core_tag_cloud( $attributes ) {
+	$class          = "wp-block-tag-cloud align{$attributes['align']}";
+	$wrapper_markup = '<p class="%1$s">%2$s</p>';
+
+	$args = array(
+		'echo'       => false,
+		'taxonomy'   => $attributes['taxonomy'],
```

Comment:
This will cause an 'Undefined index' warning when rendering a Tag Cloud block that has no taxonomy set and PHP is set to display notices. You can test this by inserting a temporary `error_reporting( E_ALL );` statement at the top of the file.

Bailing out early when `$attributes['taxonomy']` is not set is probably the best bet.

```php
if ( ! isset( $attributes['taxonomy'] ) ) {
	return '';
}
```